src/Mapbender/CoreBundle/Controller/SiteController.php

```diff
@@ -0,0 +1,82 @@
+<?php
+namespace Mapbender\CoreBundle\Controller;
+
+use Symfony\Bridge\Twig\TwigEngine;
+use Symfony\Bundle\FrameworkBundle\Controller\Controller;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpFoundation\Response;
+use Symfony\Component\HttpKernel\Exception\NotFoundHttpException;
+use Symfony\Component\Routing\Annotation\Route;
+
+
+/**
+ * Controller for site-wide URLs (!= backend, != Application-specific)
+ */
+class SiteController extends Controller
+{
+    /**
+     * Render imprint.
+     *
+     * @Route("/imprint")
+     *
+     * @param Request $request
+     * @return Response
+     */
+    public function imprintAction(Request $request)
+    {
+        $exposeImprint = $this->container->getParameter('mapbender.imprint.expose');
+        if (!$exposeImprint) {
+            throw new NotFoundHttpException();
+        }
+        $contentTemplate = $this->container->getParameter('mapbender.imprint.template.content');
+        if ($request->query->get('embed')) {
+            $template = $contentTemplate;
+            $templateVars = array();
+        } else {
+            $template = $this->container->getParameter('mapbender.imprint.template.page');
+            $templateVars = array(
+                'contentTemplate' => $contentTemplate,
+            );
+        }
+        $html = $this->getTemplatingEngine()->render($template, $templateVars);
+        return new Response($html);
+    }
+
+    /**
+     * Render data policy.
+     *
+     * @Route("/data-policy")
+     *
+     * @param Request $request
+     * @return Response
+     */
+    public function dataPolicyAction(Request $request)
```

Comment:
imprintAction and dataPolicyAction are nearly the same. Can you fold it into one function which returns the $template and $templateVars variable and takes the namespace as a parameter? Also the variable name $exposeImprint is not really fitting in the second case.